src/main/webapp/app/view/my-projects/my-projects.component.ts

```diff
@@ -22,18 +22,24 @@ import { ADMIN, SECRETARY } from 'app/shared/constants/role.constants';
   styleUrls: ['./my-projects.component.scss'],
 })
 export class MyProjectsComponent implements OnInit, OnDestroy {
-  projects: Project[][];
+  projects: Project[] = [];
+  filteredProjects: Project[][] = [];
```

Comment:
Das ist wohl kein 2d array :P

Reply:
doch, die erste Stufe ist eine Zeile (`row` der Cards), die zweite Stufe die Cards in dieser Zeile.